tests/appsec/test_versions.py

```diff
@@ -5,8 +5,9 @@
 from utils import BaseTestCase, context, interfaces, released, rfc, irrelevant, missing_feature
 
 
-@released(golang="1.36.0", java="0.90.0", nodejs="2.0.0", python="?", ruby="0.54.2")
+@released(java="0.90.0", nodejs="2.0.0", python="?", ruby="0.54.2")
 @irrelevant(library="cpp")
+@irrelevant(context.library >= "golang@1.35.0", reason="appsec span tags")
```

Comment:
Your are using appsec data in span no? If yes, you probably mean this : 

```suggestion
@irrelevant(context.library < "golang@1.35.0", reason="Before 1.35.0, go was using dedicated entry point")
```